content/_data/changelogs/lts.yml

```
@@ -8926,6 +8926,43 @@
   # pull: 7303 (PR title: Allow detached plugin location to be overridden)
   # pull: 7508 (PR title: Provide native Java 11 HTTP client versions of FormValidation#URLCheck methods)
 
+- version: "2.387.2"
+  date: 2023-03-22
+  changes:
+  - type: major bug
+    category: bug
+    pull: 7670
+    issue: 69955
+    authors:
+      - Dohbedoh
+    pr_title: "[JENKINS-69955] Make websocket connection idleTimeout configurable"
+    message: |-
+      Adjust websocket Idle Timeout to 60 seconds by default, to avoid "WebSocketTimeoutException: Connection Idle Timeout" issues.
```

I had not checked it previously, but Wikipedia uses camel case for the name of the protocol as do the [IETF documents](https://datatracker.ietf.org/doc/html/rfc6455) that describe the protocol.
```suggestion
      Adjust WebSocket idle timeout to 60 seconds by default, to avoid "WebSocketTimeoutException: Connection Idle Timeout" issues.
```